corehq/apps/smsbillables/models.py

```diff
@@ -367,3 +399,51 @@ def _get_usage_fee(cls, domain, direction):
                 % (direction, domain)
             )
         return usage_fee
+
+    @classmethod
+    def _get_twilio_charges(cls, backend_message_id, direction, couch_id):
+        def _get_twilio_client():
+            twilio_backend = SQLTwilioBackend.get_global_backends_for_this_class(SQLTwilioBackend.SMS).get()
+            config = twilio_backend.config
+            return TwilioRestClient(config.account_sid, config.auth_token)
+
+        try:
+            if backend_message_id:
+                twilio_message = _get_twilio_client().messages.get(backend_message_id)
+                return _TwilioChargeInfo(
+                    Decimal(twilio_message.price) * -1,
+                    SmsGatewayFee.get_by_criteria(
```

Comment:
(related to my previous comment) how come we still need to lookup the gateway fee?